app/packs/src/utils/viewHelpers.js

```diff
@@ -41,41 +41,52 @@ export const getMarketCapVariance = (
   tokenDayData,
   deployDate,
   startDate,
-  endDate,
   totalSupply
 ) => {
-  const supplies = [];
-  const dayData = {};
-  const dayInSeconds = 86400;
-
-  if (tokenDayData.length == 0) {
-    return '0';
+  if (startDate < deployDate) {
+    return "0%";
+  } else if (tokenDayData[0]) {
+    const startSupply = parseFloat(
+      ethers.utils.formatUnits(tokenDayData[0].dailySupply)
+    );
+    const lastSupply = parseFloat(ethers.utils.formatUnits(totalSupply));
+    const variance = (lastSupply - startSupply) / startSupply;
+    return `${variance < 0 ? "" : "+"}${parseAndCommify(variance)}%`;
+  } else {
+    return "0%";
```

Comment:
Small improvement to reduce code branching.

```suggestion
  if (startDate < deployDate || !tokenDayData[0]) {
    return "0%";
  } 
  
  const startSupply = parseFloat(
    ethers.utils.formatUnits(tokenDayData[0].dailySupply)
  );
  const lastSupply = parseFloat(ethers.utils.formatUnits(totalSupply));
  const variance = (lastSupply - startSupply) / startSupply;
  return `${variance < 0 ? "" : "+"}${parseAndCommify(variance)}%`;
```